.github/workflows/backport.yml

```diff
@@ -23,16 +23,17 @@ jobs:
       - name: Create pull request
         env:
           NUMBER: ${{ github.event.inputs.number }}
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          GITHUB_TOKEN: ${{ secrets.BOT_TOKEN }}
```

Comment:
do we have this BOT_TOKEN? I don't see it in our secrets settings.

Reply:
ya, see above:

> I added "opentelemetry-java-bot PAT" to the 1password account, will need to create a github action secret named `BOT_TOKEN`.

Reply:
ah, ok. We should do that before we merge, correct?

Reply:
technically need to do it before the next release, but ya, better to do it now before we forget 😄

Reply:
ok. I'll go ahead and do that now

Reply:
done